packages/notte-sdk/src/notte_sdk/endpoints/sessions.py

```
@@ -663,6 +673,12 @@ def stop(self) -> None:
             ValueError: If the session hasn't been started (no session_id available).
             RuntimeError: If the session fails to close properly.
         """
+        if self._cookie_file is not None:
+            try:
+                cookies = self.get_cookies()
+                create_or_append_cookies_to_file(self._cookie_file, cookies)
+            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)